modules/tensor_mechanics/include/materials/ViscoplasticityStressUpdateBase.h

```diff
@@ -118,7 +118,9 @@ class ViscoplasticityStressUpdateBaseTempl : public StressUpdateBaseTempl<is_ad>
   using ViscoplasticityStressUpdateBaseTempl<is_ad>::_porosity_old;                                \
   using ViscoplasticityStressUpdateBaseTempl<is_ad>::updateIntermediatePorosity;                   \
   using ViscoplasticityStressUpdateBaseTempl<is_ad>::computeStressFinalize;                        \
-  using ViscoplasticityStressUpdateBaseTempl<is_ad>::computeStressInitialize;
+  using ViscoplasticityStressUpdateBaseTempl<is_ad>::computeStressInitialize;                      \
+  using ViscoplasticityStressUpdateBaseTempl<is_ad>::computeResidual;                              \
```

Comment:
Whoops! computeResidual and computeDerivative don't actually live in `ViscoplasticityStressUpdateBase.h`...will remove